cmd/ndm_daemonset/probe/eventhandler.go

```diff
@@ -102,11 +102,16 @@ func (pe *ProbeEvent) addDiskEvent(msg controller.EventMessage) {
 	}
 }
 
-// deleteDiskEvent deactivate disk resource using uuid from etcd
+// deleteDiskEvent deactivate disk/blockdevice resource using uuid from etcd
```

Comment:
@akhilerm -- how about splitting this function into two - for block device and disk?

Reply:
@kmova . The function is called when we receive a udev Remove Event. At that point, we do not have a Disk/BlockDevice separation.